TheCodeBlog/AppDelegate.swift

```diff
@@ -14,7 +14,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
```

Comment:
<!--
  1 failure:  Line should be 12...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: TheCodeBlog/AppDelegate.swift;
  Line: 16;
-->
- :no_entry_sign: Line should be 120 characters or less: currently 145 characters (`line_length`)